module/library/categories/helper.js

```diff
@@ -173,6 +174,94 @@ module.exports = class LibraryCategoriesHelper {
 					})
 				}
 
+				let projectTemplates = result[0].data
+				let allCategoryId = []
+				let filePathsArray = []
+
+				for (let project of projectTemplates) {
+					let categories = project.categories
+					if (categories.length > 0) {
+						let categoryIdArray = categories.map((category) => {
+							if (category._id) {
+								return category._id
+							}
+						})
+						allCategoryId.push(...categoryIdArray)
+					}
+				}
+
+				let allCategoryInfo = await projectCategoriesQueries.categoryDocuments({
+					_id: { $in: allCategoryId },
+				})
+
+				for (let singleCategoryInfo of allCategoryInfo) {
+					if (singleCategoryInfo.evidences && singleCategoryInfo.evidences.length > 0) {
+						let filePaths = singleCategoryInfo.evidences.map((evidenceInfo) => {
+							return evidenceInfo.filepath
+						})
+
+						filePathsArray.push({
+							categoryId: singleCategoryInfo._id,
+							filePaths,
+						})
+					}
+				}
+
+				for (let project of projectTemplates) {
+					let categories = project.categories
+
+					if (categories.length > 0) {
+						for (let projectCategory of categories) {
+							let filteredCategory = allCategoryInfo.filter((category) => {
+								return category._id.toString() == projectCategory._id.toString()
+							})
+
+							let singleCategoryInfo = filteredCategory[0]
+							projectCategory.evidences = singleCategoryInfo.evidences
+						}
+					}
+				}
+
+				let allFilePaths = filePathsArray.map((project) => {
+					return project.filePaths
+				})
+
+				let finalArr = _.flatten(allFilePaths)
```

Comment:
@borkarsaish65 please rename the variable to more meaning full word, Please add some comments